vector/src/main/java/im/vector/app/AppStateHandler.kt

```diff
@@ -125,11 +133,28 @@ class AppStateHandler @Inject constructor(
                         } else {
                             setCurrentGroup(uiStateRepository.getSelectedGroup(session.sessionId), session)
                         }
+                        observeSyncStatus(session)
                     }
                 }
                 .launchIn(coroutineScope)
     }
 
+    private fun observeSyncStatus(session: Session) {
+        session.getSyncStatusLive()
+                .asFlow()
+                .filterIsInstance<SyncStatusService.Status.IncrementalSyncDone>()
```

Comment:
we can use `distinct` to avoid keeping track of the previous state https://github.com/vector-im/element-android/pull/5300/files#diff-d5584663de96dcd881e4441839a2a6e44b50a58268b0b9af4d17be21fba9897fR152

Reply:
👍  added one line below